VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts

```diff
@@ -123,6 +124,7 @@ export class JudgeParticipantStatusListComponent extends WRParticipantStatusList
 
     async saveJudgeDisplayName() {
         this.judge.display_name = this.newJudgeDisplayName;
+        this.judge.display_name = new SafePipe().transform(this.judge.display_name);
```

Comment:
Pipes should be used in templates. If it doesn't need to be used in a template, like this
```
{ someValue | safe }
```
then it doesn't need to be a pipe & should just be a func.

Reply:
Pipe removed